src/lib/tdtool.js

```diff
@@ -48,6 +48,9 @@ const TDtool = {
   run: (cmd, target) => TDtool.isInstalled().then(() =>
     execute(`tdtool ${cmd} ${target}`)),
 
+ run: (cmd1, target1, cmd2, target2) => TDtool.isInstalled().then(() =>
```

Comment:
This actually overrides the previous `run` on line 48, which might/will cause unexpected behaviour.